File: distribution/src/main/java/com/music/distribution/entity/Music.java
package com.music.distribution.entity;

import jakarta.persistence.*;
import lombok.*;

import java.util.Date;

@Entity
@Data
@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Music {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Long id;
    public String title;
    @Lob
    @Column(length = 1000)
    public String description;
    public String link;
    public String image;
    public Date createdAt;
    public Date pubDate;
}

File: music-reader/src/app/class/music/music.ts
export class Music {
    id: number;
    title: string;
    description: string;
    link: string;
    image: string;
    createdAt: Date;
    pubDate: Date;
  
    constructor(id: number, title: string, description: string, link: string, image: string, createdAt: Date, pubDate: Date) {
      this.id = id;
      this.title = title;
      this.description = description;
      this.link = link;
      this.image = image;
      this.createdAt = createdAt;
      this.pubDate = pubDate;
    }
  }
  